libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -145,106 +155,155 @@ export const NavPaperWrapper = styled('div')`
   display: grid;
-  gap: ${({theme}) => theme.spacing(3)};
+  gap: ${({theme}) => theme.spacing(1)};
+  grid-auto-rows: max-content;
+`
+
+export const Name = styled('span')`
+  text-transform: uppercase;
+  font-weight: 300;
+  font-size: 14px;
+`
+export const Separator = styled('div')`
+  width: 100%;
+  height: 1px;
+  background-color: white;
 
   ${({theme}) => css`
     ${theme.breakpoints.up('sm')} {
-      grid-auto-flow: column;
-      grid-auto-columns: max-content;
+      display: none;
     }
   `}
 `
 
-export const NavPaperCategory = styled('div')`
+export const LinksGroup = styled('div')`
   display: grid;
-  gap: ${({theme}) => theme.spacing(2)};
-  grid-auto-rows: max-content;
+  grid-template-columns: 1fr;
+  gap: ${({theme}) => theme.spacing(3)};
+
+  ${({theme}) => css`
+    ${theme.breakpoints.up('sm')} {
+      grid-template-columns: 1fr 1fr;
+    }
+  `}
 `
 
 const navPaperLinkStyling = (theme: Theme) => css`
-  border-bottom: 2px solid currentColor;
-
   ${theme.breakpoints.up('sm')} {
     border-bottom: 0;
   }
 `
 
 export const NavPaperCategoryLinks = styled('div')`
   display: grid;
-  gap: ${({theme}) => theme.spacing(2)};
-  text-transform: uppercase;
   font-weight: ${({theme}) => theme.typography.fontWeightMedium};
   font-size: ${({theme}) => theme.typography.h6.fontSize};
 `
 
 export const NavPaperMainLinks = styled(NavPaperCategoryLinks)`
+  gap: ${({theme}) => theme.spacing(1)};
+`
+
+export const NavPaperChildrenWrapper = styled('div')`
+  position: relative;
+  padding: ${({theme}) => theme.spacing(1.5)};
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(40px, 1fr));
+  justify-items: center;
+  left: 0;
+  top: 0;
+  width: 100%;
+
   ${({theme}) => css`
-    ${theme.breakpoints.up('sm')} {
-      display: none;
+    ${theme.breakpoints.up('md')} {
+      position: absolute;
+      grid-template-columns: auto;
+      justify-items: start;
+      width: calc(100% / 6);
+      gap: ${theme.spacing(3)};
+      padding-top: ${theme.spacing(10)};
+      padding-left: ${theme.spacing(4.5)};
     }
   `}
 `
 
 const NavPaper = ({
   main,
   categories,
-  closeMenu
+  closeMenu,
+  children
 }: {
   main: FullNavigationFragment | null | undefined
-  categories: FullNavigationFragment[]
+  categories: FullNavigationFragment[][]
   closeMenu: () => void
+  children: React.ReactNode
 }) => {
   const {
-    elements: {Link, H5}
+    elements: {Link, H4, H6}
   } = useWebsiteBuilder()
   const theme = useTheme()
 
   return (
     <NavPaperWrapper>
+      {children && <NavPaperChildrenWrapper>{children}</NavPaperChildrenWrapper>}
       {!!main?.links.length && (
         <NavPaperMainLinks>
           {main.links.map((link, index) => {
             const url = navigationLinkToUrl(link)
 
             return (
               <Link href={url} key={index} color="inherit" underline="none" onClick={closeMenu}>
-                {link.label}
+                <H4>{link.label}</H4>
```

Comment:
a11y problem as this renders `<h4>`, give it a more appropriate element to render